domain/src/test/java/org/oppia/android/domain/classify/rules/textinput/TextInputContainsRuleClassifierProviderTest.kt

```diff
@@ -24,6 +24,29 @@ import kotlin.test.fail
 @Config(manifest = Config.NONE)
 class TextInputContainsRuleClassifierProviderTest {
 
+  private val STRING_INPUT_CONTAINS_ANSWER =
```

Comment:
Other two files contain the correct variable naming like `STRING_ANSWER_TEST_UPPERCASE`. 
Let's use the same here. 

`STRING_VALUE_TEST_CONTAINS_ANSWER `

Reply:
Done